code/game/gamemodes/machinegod/sigils.dm

```diff
@@ -0,0 +1,130 @@
+/obj/effect/sigil
+	name = "sigil"
+	desc = "An odd circle."
+	anchored = 1
+	icon = 'icons/obj/clockwork/sigils.dmi'
+	icon_state = "ritual"
+	color = "#B39700"
+	var/actcolor = ""
+	var/visibility = 0
+	unacidable = 1
+	layer = TURF_LAYER
+
+	var/dead=0 // For cascade and whatnot.
+
+	var/atom/movable/overlay/h_animation = null
+	var/nullblock = 0
+	var/culttrigger = 0 //Can Ratvar cultists trigger this?
+
+/obj/effect/sigil/Crossed(AM as mob|obj)
+	Bumped(AM)
+
+/obj/effect/sigil/Bumped(mob/M as mob|obj)
+	if(istype(M, /mob/living/carbon/human) || istype(M, /mob/living/carbon/monkey) || istype(M, /mob/living/silicon))
+		activation(M)
+		/*if(isclockcult(M)) NOT IMPLEMENTED YET -velardamakar
+			if(culttrigger)
+				activation(M)
+		else
+			activation(M)*/
+
+/obj/effect/sigil/cultify() //PURGE
+	qdel(src)
+
+/obj/effect/sigil/proc/activation(var/mob/M as mob) //What does it do when it's triggered?
+	if(M.stat==DEAD)
```

Comment:
Should probably be just ```if(M.stat)``` since I dont think you want unconscious people to use it.